README.md

```diff
@@ -359,6 +359,17 @@ You will have to add the serialization, date_time, and regex modules to Boost wh
 
 To build: `env CC=egcc CXX=eg++ CPP=ecpp DEVELOPER_LOCAL_TOOLS=1 BOOST_ROOT=/path/to/the/boost/you/built make release-static-64`
 
+### On Solaris:
+
+The default Solaris linker can't be used, you have to install GNU ld, then run cmake manually with the path to your copy of GNU ld:
+
+        mkdir -p build/release
+        cd build/release
+        cmake -DCMAKE_LINKER=/path/to/ld -D CMAKE_BUILD_TYPE=Release ../..
```

Comment:
Can we set CMAKE_LINKER from within CMakeLists.txt?

Reply:
Not really, since GNU ld is not installed by default.
Unless we probe well known paths and run with --version and parse, but that seems meh.